src/System.Data.SqlClient/src/Configurations.props

```diff
@@ -12,9 +12,9 @@
       netcoreapp-Unix;
       netcoreapp-Windows_NT;
       uap10.0.16299-Windows_NT;
+      uap-Windows_NT;
```

Comment:
What is the reason for this change?  It looks to me like the UAP build and uap10.0.16299 build are identical, I don't see a good reason to put both in the package.